### ISSUE TITLE
after flutter 1.17.0 update not working in android (not sure in ios)

### ISSUE THREAD

**Issue by mitesh77** [razorpay/razorpay-flutter#77]
check with creating new project too same error face

output error :
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':razorpay_flutter'.
> Could not open cp_proj remapped class cache for eii7mfeilb85f6s74q7cms80e (C:\Users\Mitesh\.gradle\caches\5.6.2\scripts-remapped\build_cjs4r0rudwznpdupsjx0zx6qp\eii7mfeilb85f6s74q7cms80e\cp_proj40e7ec637808a9c747559d26950b1d9b).
   > Could not open cp_proj generic class cache for build file 'C:\Flutter\flutter\.pub-cache\hosted\pub.dartlang.org\razorpay_flutter-1.1.4\android\build.gradle' (C:\Users\Mitesh\.gradle\caches\5.6.2\scripts\eii7mfeilb85f6s74q7cms80e\cp_proj\cp_proj40e7ec637808a9c747559d26950b1d9b).
      > Build file 'C:\Flutter\flutter\.pub-cache\hosted\pub.dartlang.org\razorpay_flutter-1.1.4\android\build.gradle' should not contain a package statement.
> Could not get unknown property 'android' for project ':razorpay_flutter' of type org.gradle.api.Project.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
Exception: Gradle task assembleDebug failed with exit code 1

**Comment by ajsatam** [comment]
I too am getting the same error. Please provide the solution.

**Comment by SoorajSoman** [comment]
Duplicate of #74

**Comment by kaumudpa** [comment]
Not working on ios as well.
Here is the output. 
```
Xcode's output:
↳
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    /Users/kaumudpachauri/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.4/ios/Classes/RazorpayDelegate.swift:4:42: error:
    'RazorpayPaymentCompletionProtocolWithData' is unavailable: cannot find Swift declaration for this protocol
    public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Razorpay.RazorpayPaymentCompletionProtocolWithData:2:17: note: 'RazorpayPaymentCompletionProtocolWithData' has been explicitly marked unavailable here
    public protocol RazorpayPaymentCompletionProtocolWithData {
                    ^
    /Users/kaumudpachauri/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.4/ios/Classes/RazorpayDelegate.swift:4:85: error:
    'ExternalWalletSelectionProtocol' is unavailable: cannot find Swift declaration for this protocol
    public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
                                                                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    Razorpay.ExternalWalletSelectionProtocol:2:17: note: 'ExternalWalletSelectionProtocol' has been explicitly marked unavailable here
    public protocol ExternalWalletSelectionProtocol {
                    ^
    /Users/kaumudpachauri/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.4/ios/Classes/RazorpayDelegate.swift:57:24: error: use of unresolved
    identifier 'RazorpayCheckout'
            let razorpay = RazorpayCheckout.initWithKey(key ?? "", andDelegateWithData: self)
                           ^~~~~~~~~~~~~~~~
    Command CompileSwift failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Could not build the precompiled application for the device.
```

This needs an urgent fix.

**Comment by PritishSawant** [comment]
> Not working on ios as well.
> Here is the output.
> Flutter Stable : 1.17.0
> Xcode : 11.4.1
> 
> ```
> Xcode's output:
> ↳
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     /Users/kaumudpachauri/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.4/ios/Classes/RazorpayDelegate.swift:4:42: error:
>     'RazorpayPaymentCompletionProtocolWithData' is unavailable: cannot find Swift declaration for this protocol
>     public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
>                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     Razorpay.RazorpayPaymentCompletionProtocolWithData:2:17: note: 'RazorpayPaymentCompletionProtocolWithData' has been explicitly marked unavailable here
>     public protocol RazorpayPaymentCompletionProtocolWithData {
>                     ^
>     /Users/kaumudpachauri/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.4/ios/Classes/RazorpayDelegate.swift:4:85: error:
>     'ExternalWalletSelectionProtocol' is unavailable: cannot find Swift declaration for this protocol
>     public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
>                                                                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     Razorpay.ExternalWalletSelectionProtocol:2:17: note: 'ExternalWalletSelectionProtocol' has been explicitly marked unavailable here
>     public protocol ExternalWalletSelectionProtocol {
>                     ^
>     /Users/kaumudpachauri/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.4/ios/Classes/RazorpayDelegate.swift:57:24: error: use of unresolved
>     identifier 'RazorpayCheckout'
>             let razorpay = RazorpayCheckout.initWithKey(key ?? "", andDelegateWithData: self)
>                            ^~~~~~~~~~~~~~~~
>     Command CompileSwift failed with a nonzero exit code
>     note: Using new build system
>     note: Building targets in parallel
>     note: Planning build
>     note: Constructing build description
> 
> Could not build the precompiled application for the device.
> ```
> 
> This needs an urgent fix.

Seems to be working for me on iOS. I am facing issue on Android

**Comment by sumedht** [comment]
@mitesh77 please do above changes
`Go to flutterSdk folder->.pub-cache->hosted->pub.dartlang.org-> razorpay_flutter-1.1.4->android
open the build.gradle , it is the same file in this repo razorpay-flutter->android->build.gradle
You just need to remove the first line package android`

**Comment by mitesh77** [comment]
@sumedht  it working thanks but this is just a temporary solution.

**Comment by sumedht** [comment]
@mitesh77 yes. we will fix in next update

**Comment by kaumudpa** [comment]
@sumedht This issue was closed without addressing the ios issue.

**Comment by SandyAra** [comment]
> Not working on ios as well.
> Here is the output.
> Flutter Stable : 1.17.0
> Xcode : 11.4.1
> 
> ```
> Xcode's output:
> ↳
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     Command CompileSwift failed with a nonzero exit code
>     /Users/kaumudpachauri/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.4/ios/Classes/RazorpayDelegate.swift:4:42: error:
>     'RazorpayPaymentCompletionProtocolWithData' is unavailable: cannot find Swift declaration for this protocol
>     public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
>                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     Razorpay.RazorpayPaymentCompletionProtocolWithData:2:17: note: 'RazorpayPaymentCompletionProtocolWithData' has been explicitly marked unavailable here
>     public protocol RazorpayPaymentCompletionProtocolWithData {
>                     ^
>     /Users/kaumudpachauri/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.4/ios/Classes/RazorpayDelegate.swift:4:85: error:
>     'ExternalWalletSelectionProtocol' is unavailable: cannot find Swift declaration for this protocol
>     public class RazorpayDelegate: NSObject, RazorpayPaymentCompletionProtocolWithData, ExternalWalletSelectionProtocol {
>                                                                                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     Razorpay.ExternalWalletSelectionProtocol:2:17: note: 'ExternalWalletSelectionProtocol' has been explicitly marked unavailable here
>     public protocol ExternalWalletSelectionProtocol {
>                     ^
>     /Users/kaumudpachauri/.pub-cache/hosted/pub.dartlang.org/razorpay_flutter-1.1.4/ios/Classes/RazorpayDelegate.swift:57:24: error: use of unresolved
>     identifier 'RazorpayCheckout'
>             let razorpay = RazorpayCheckout.initWithKey(key ?? "", andDelegateWithData: self)
>                            ^~~~~~~~~~~~~~~~
>     Command CompileSwift failed with a nonzero exit code
>     note: Using new build system
>     note: Building targets in parallel
>     note: Planning build
>     note: Constructing build description
> 
> Could not build the precompiled application for the device.
> ```
> 
> This needs an urgent fix.

same issue for me

**Comment by srivastavarnav** [comment]
Any update on this?
